doc/install.txt

```diff
@@ -4,891 +4,17 @@
-  contain
-
-    .. code-block:: cfg
-
-      [cmodule]
-      mac_framework_link=True
-
-- More generally, to investigate libraries issues, you can use the ``otool -L``
-  command on ``.so`` files found under your ``~/.theano`` directory. This will
-  list shared libraries dependencies, and may help identify incompatibilities.
-
-Please inform us if you have trouble installing and running Theano on your Mac.
-We would be especially interested in dependencies that we missed listing,
-alternate installation steps, GPU instructions, as well as tests that fail on
-your platform (use the ``theano-users@googlegroups.com`` mailing list, but
-note that you must first register to it, by going to `theano-users`_).
-
-
-Windows
--------
-
-:ref:`install_windows` provides step-by-step instructions on how to install Theano on 32- or 64-bit Windows systems, using freely available
-tools and compilers.
-
-Editing code in Visual Studio
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-You will find a Visual Studio solution file (``Theano.sln``) in the root of
-the Theano repository. Note that this project file may not be kept up-to-date
-and is not officially supported by the core Theano developers: it is provided
-for convenience only.
-Also, be aware that it will not make Theano use Visual Studio to compile C
-files: it is only meant to provide an easy way to edit Theano code within
-the Visual Studio editor.
-
-
-Windows Installation References
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-1. http://stackoverflow.com/questions/9047072/windows-python-version-and-vc-redistributable-version
-2. http://stackoverflow.com/questions/1865069/how-to-compile-a-64-bit-application-using-visual-c-2010-express
-3. http://blog.victorjabur.com/2011/06/05/compiling-python-2-7-modules-on-windows-32-and-64-using-msvc-2008-express/
-4. http://stackoverflow.com/questions/126279/c99-stdint-h-header-and-ms-visual-studio
-5. http://stackoverflow.com/questions/11182765/how-can-i-build-my-c-extensions-with-mingw-w64-in-python
-6. https://mail.python.org/pipermail/python-announce-list/2014-September/010457.html
-
-
-Generating the documentation
-----------------------------
-
-You can read the latest HTML documentation `here
-<http://deeplearning.net/software/theano>`__.
-You can download the latest PDF documentation `here
-<http://deeplearning.net/software/theano/theano.pdf>`__.
-
-We recommend you look at the documentation on the website, since it
-will be more current than the documentation included with the package.
-
-If you really wish to build the documentation yourself, you will need
-sphinx, as described above. Issue the following command::
-
-    python ./doc/scripts/docgen.py
-
-Documentation is built into ``html/``.
-The PDF of the documentation is ``html/theano.pdf``.
-
-
-.. _theano-users: http://groups.google.com/group/theano-users?pli=1
-.. _theano-dev: http://groups.google.com/group/theano-dev?pli=1
+If you are curious about what Theano requires, you can head up
```

Comment:
The point was exactly preventing the user from installing the requirements by himself: pip will install all the dependencies for you in most of the cases, so the prerequisites page is more for people who for some reason want to have control on what they install, rather than for the normal user. Does it make sense?